nodeJS/APIs/API-Security.md

```diff
@@ -25,6 +25,15 @@ This strategy, while particularly useful with APIs can be used with a traditiona
 2. [This article](https://dev.to/_arpy/learn-using-jwt-with-passport-authentication-22n8) covers setting up JSON Web Tokens within the PassportJS system that you should already be familiar with. In this case you don't _really_ need Passport, but it does handle quite a bit of stuff that the guy in the video set up manually.
 </div>
 
+
+### Knowledge Checks 
+This section contains questions for you to check your understanding of this lesson. If you’re having trouble answering the questions below on your own, review the material above to find the answer.
+
+- <a class='knowledge-check-link' href='https://dev.to/_arpy/learn-using-jwt-with-passport-authentication-22n8'>What is a JSON web token?</a>
+- <a class='knowledge-check-link' href='#securetoken'>What are two things a secure token will do?</a>
+- <a class='knowledge-check-link' href='#passtoken'>Where in the code is a secure token passed?</a>
```

Comment:
I would change these ids (and their corresponding spans of course) to separate the words with a hyphen.

Reply:
Done. Good idea, thanks @wise-king-sullyman